sambacc/config.py

```diff
@@ -52,11 +52,19 @@ class ValidationUnsupported(Exception):
     pass
 
 
+class _FakeRefResolutionError(Exception):
+    pass
+
+
 def _schema_validate(data: dict[str, typing.Any], version: str) -> None:
     try:
         import jsonschema  # type: ignore[import]
     except ImportError:
         raise ValidationUnsupported()
+    try:
+        _refreserror = getattr(jsonschema, "RefResolutionError")
+    except AttributeError:
+        _refreserror = _FakeRefResolutionError
```

Comment:
`refreserror`

Is that short for `refresherror`?

Reply:
`Ref`erence `Res`olution `error`